sample/src/androidTest/java/com/schibsted/spain/barista/sample/ScrollsTest.java

```diff
@@ -23,36 +23,58 @@
   public FailureHandlerValidatorRule handlerValidator = new FailureHandlerValidatorRule();
 
   @Test
-  public void scrollsInsideScrollable() {
+  public void scrollsInsideScrollable_byId() {
     assertTopVisible();
-
     scrollTo(R.id.really_far_away_button);
+    assertBottomVisible();
```

Comment:
The blank lines are there to separate the given/when/then parts of the tests. I think they help distinguish that the important method is is `scrollTo`

Reply:
Sorry, I thought in the rule we follow when we worked together, about avoiding to add blank lines when a test just has one line for given, another one for the when and one more for the then.

Reply:
Um your memory is better than mine, so you're probably right n_n' 
Still, I feel like in this case I read it better with blank lines.

Whatever, thanks for changing it!